contents/handbook/engineering/developing-locally.md

```diff
@@ -437,3 +437,77 @@ With PyCharm's built in support for Django, it's fairly easy to setup debugging
 ## Extra: Developing paid features (PostHog employees only)
 
 If you're a PostHog employee, you can get access to paid features on your local instance to make development easier. [Learn how to do so in our internal guide](https://github.com/PostHog/billing?tab=readme-ov-file#licensing-your-local-instance).
+
+## Extra: Working with the data warehouse and a MySQL source
+
+If you want to set up a local MySQL database as a source for the data warehouse, there are a few extra set up steps you'll need to complete:
+1. Setting up a local MySQL database to connect to.
+2. Installing MS SQL drivers on your machine.
+3. Defining additional environment variables for the Temporal task runner.
+
+First, install MySQL:
+
+```bash
+brew install mysql
+brew services start mysql
+```
+
+Once MySQL is installed, create a database and table, insert a row, and create a user who can connect to it:
+
+```bash
+mysql -u root
+```
+```sql
+CREATE DATABASE posthog_dw_test;
+CREATE TABLE IF NOT EXISTS payments (id INT AUTO_INCREMENT PRIMARY KEY, timestamp DATETIME, distinct_id VARCHAR(255), amount DECIMAL(10,2));
+INSERT INTO payments (timestamp, distinct_id, amount) VALUES (NOW(), 'testuser@example.com', 99.99);
+CREATE USER 'posthog'@'%' IDENTIFIED BY 'posthog';
+GRANT ALL PRIVILEGES ON posthog_dw_test.* TO 'posthog'@'%';
+FLUSH PRIVILEGES;
+```
+
+Next, you'll need to install some MS SQL drivers for PostHog the application to connect to the MySQL database:
+
+```bash
+brew tap microsoft/mssql-release https://github.com/Microsoft/homebrew-mssql-release
+brew update
+HOMEBREW_ACCEPT_EULA=Y brew install msodbcsql18 mssql-tools18 FreeTDS
+```
```

Comment:
We have a bunch of this listed here: https://github.com/PostHog/posthog/blob/master/posthog/warehouse/README.md - It may be worth linking to that as opposed to maintaining two copies of the same thing, one will bound to get out of date

Reply:
@Gilbert09 Would you prefer to roll that document into this one, or should I remove the detail here and link to that one?

Reply:
I reckon remove the detail here and link to the one in the `posthog` repo - that one is closer to the code and likely will be updated etc

Reply:
Sounds good, updated in c6fddf33d